src/api/store/graph.py

```diff
@@ -393,3 +405,44 @@ def list_graphs_for_super_admin(self, context: Context):
     except Exception as e:
       capture_message(str(e), level="error")
       return None, CustomMassenergizeError(str(e))
+
+  def debug_data_fix(self) -> None:
+    # attempting to fix the problem with data getting screwed up
```

Comment:
I would love a documentation on what the problem is and what approach we are taking to solve it.  Its not very clear why we are doing what we are doing here.

Reply:
API issue #448 documents this (https://github.com/massenergize/api/issues/448)
The reported values in the Data model get set to many thousands, so the impact page looks crazy.
We have observed this several times.  The debug_data_fix code is what I use to automatically fix it by running the portal locally, it takes a few seconds.  Works sort of like the data.backfill but much faster.